lstmcpipe/scripts/script_batch_filelist_lst.py

```diff
@@ -54,7 +54,8 @@ def main():
             if args.config_file:
                 cmd.append("--config={}".format(args.config_file))
 
-            subprocess.run(cmd)
+            outfile = Path(args.output_dir).joinpath('dl1_' + file.replace('.simtel.gz', '.h5'))
```

Comment:
We are doing a loop over a list of files within the a `*.sublist` file. you can directly recover `outfile` as `file`

Reply:
`file` is the input file here
`rerun_cmd` needs the  output file

Reply:
ah, right.

Reply:
`file` is an str absolute path , though. You would need to do Path(file).name to recover just the file name

Reply:
thanks, fixed